src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,69 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { CompletionFilter } from '../../types/CompletionFilter';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  searchQuery: string,
+  changeSearchQuery: (value: string) => void,
+  completionStatus: CompletionFilter,
+  changeCompletionStatus: (value: CompletionFilter) => void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  searchQuery,
+  changeSearchQuery,
+  completionStatus,
+  changeCompletionStatus,
+}) => {
+  const capitalizeFirstLetter = (string: string) => {
+    return string[0].toUpperCase() + string.slice(1);
+  };
```

Comment:
In CSS we have `text-transform: capitalize;` so you can use it instead of capitalizing with js:
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Also, if you create helper functions move them outside or to utils/helpers folder